src/status_im/browser/core.cljs

```diff
@@ -239,24 +233,30 @@
 
 (fx/defn navigation-state-changed
   [cofx event error?]
-  (let [{:strs [url loading title]} (js->clj event)]
-    (fx/merge cofx
-              (update-browser-option :loading? loading)
-              (update-browser-name title)
-              (update-browser-on-nav-change url error?))))
+  (let [{:strs [url loading title]} (js->clj event)
+        deep-link? (utils.universal-links/deep-link? url)]
+    (if (utils.universal-links/universal-link? url)
+      (if (and deep-link? platform/ios?) ;; ios webview handles this
+        (fx/merge cofx {})
```

Comment:
seems like this does nothing? so you could replace the `if` by a `when-not` and remove that line